unified_targets/configs/FURYF4OSD.config

```diff
@@ -0,0 +1,122 @@
+# Betaflight / STM32F405 (S405) 4.1.0 May 11 2019 / 13:09:51 (a8e9dd94e) MSP API: 1.42
+
+board_name FURYF4OSD
+manufacturer_id DIAT
+
+# resources
+resource BEEPER 1 A08
+resource MOTOR 1 A03
+resource MOTOR 2 B00
+resource MOTOR 3 B01
+resource MOTOR 4 A02
+resource PPM 1 C09
+resource LED_STRIP 1 A00
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 6 C07
+resource INVERTER 1 C00
+resource I2C_SCL 1 B06
+resource I2C_SDA 1 B07
+resource LED 1 B05
+resource LED 2 B04
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 C10
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 C11
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 C12
+resource ESCSERIAL 1 C09
+resource CAMERA_CONTROL 1 B09
+resource ADC_BATT 1 C01
+resource ADC_RSSI 1 C02
+resource ADC_CURR 1 C03
+resource FLASH_CS 1 B03
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+resource USB_DETECT 1 C05
+
+# timer
+timer C09 1
+timer A03 0
+timer B00 1
+timer B01 0
+timer A02 0
+timer A00 1
+timer B09 1
+
+# dma
+dma ADC 1 1
+# ADC 1: DMA2 Stream 4 Channel 0
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+dma pin A03 1
+# pin A03: DMA1 Stream 6 Channel 3
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA2 Stream 6 Channel 0
+dma pin A02 0
+# pin A02: DMA1 Stream 1 Channel 3
+dma pin A00 0
+# pin A00: DMA1 Stream 2 Channel 6
+
+# feature
+feature -RX_PARALLEL_PWM
+feature -AIRMODE
+feature RX_SERIAL
+feature OSD
+
+# beacon
+beacon RX_LOST
+

```

Comment:
Please remove non-hardware settings from this.

Reply:
From section 5,

> set all the custom settings that are specific to your board (e.g. presets for serial RX on a specific port if the board's instructions are for users to use this port for the serial RX). Changes can be done in the UI or in CLI;

Reply:
Can you please point me to the manufacturer's instructions that tell the user to configure an aux channel (for example) on their TX?